### PR TITLE
Change to SIGNALFX_ env vars

### PR DESCRIPTION
**Pull request by RassK**
* Update env vars to SIGNALFX_*
* fix log paths

### CODE REVIEW

**Review by pjanotti — APPROVED** [review]
Let's just wait on the definition about which prefix we should actually use.

File: .azure-pipelines/setup_tracer.ps1 [hunk 1]
```diff
@@ -2,81 +2,81 @@ $ProgressPreference = 'SilentlyContinue'
 
 echo "Getting latest release version"
 # Get the latest release tag from the github release page
-$release_version = (Invoke-WebRequest https://api.github.com/repos/datadog/dd-trace-dotnet/releases | ConvertFrom-Json)[0].tag_name.SubString(1)
+$release_version = (Invoke-WebRequest https://api.github.com/repos/signalfx/signalfx-dotnet-tracing/releases | ConvertFrom-Json)[0].tag_name.SubString(1)
```
[the remark this answers is not in the document]
Comment (RassK) [reply .azure-pipelines/setup_tracer.ps1:5]: Some similar issues with `Build.Github.cs` `Build.Gitlab.cs`

**Comment by pellared** [comment]
I have found some occurrences like these in dockerfiles:

```
ENV CORECLR_PROFILER_PATH=/opt/datadog/OpenTelemetry.AutoInstrumentation.ClrProfiler.Native.so
ENV SIGNALFX_INTEGRATIONS=/opt/datadog/integrations.json
ENV SIGNALFX_DOTNET_TRACER_HOME=/opt/datadog
```

**Comment by pellared** [comment]
In `pal.h`

```
        return ToWSTRING(program_data +
            R"(\Datadog .NET Tracer\logs\dotnet-tracer-native.log)");
#else
        return WStr("/var/log/datadog/dotnet/dotnet-tracer-native.log");
#endif
```

**Comment by pellared** [comment]
Can you update the modified files with `// Modified by Splunk Inc.` comment for all modified files?

**Comment by pjanotti** [comment]
> Included a quick manual PS tool script copyright.ps1

Let's improve it as we go.

**Review by pjanotti — APPROVED** [review]
Looks good as far as I can see :) Likely, we missed some spots but will catch up on those as needed.

**Pull request merged**